presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/ControllerServer.java

```diff
@@ -33,16 +36,21 @@ public static void main(String[] args)
     {
         // TODO: Replace placeholder properties with proper configuration
         System.setProperty("node.environment", "test");
-        System.setProperty("http-server.http.port", "8081");
+        System.setProperty("http-server.http.port", "8088");
         System.setProperty("log.levels-file", "etc/log.properties");
+        System.setProperty("discovery.uri", "http://localhost:"
+                + System.getProperty("http-server.http.port"));
 
         Bootstrap bootstrap = new Bootstrap(
                 new NodeModule(),
                 new DiscoveryModule(),
+                new DiscoveryServerModule(),
+                new HttpEventModule(),
                 new HttpServerModule(),
                 new JsonModule(),
-                new JaxrsModule(true), // requireExplicitBindings = true
-                new HttpEventModule(),
+                new JaxrsModule(true), // requireExplicitBindings = true'
+                new JmxModule(),
+                new MBeanModule(),
```

Comment:
The order of the modules is sort-of arbitrary. The Node and Discovery modules are first, as in the airlift samples and Presto, then the build-in modules in alphabetical order, then the single custom module.